examples/custom-receiver/src/KoaReceiver.ts

```diff
@@ -0,0 +1,335 @@
+/* eslint-disable node/no-extraneous-import */
```

Comment:
I'm going to add one more example (perhaps, fastify) here. I'm planning that these receivers can be eventually released as optional npm packages.